src/query/sensor/sensor.module.ts

```diff
@@ -44,8 +44,12 @@ export class SensorQueryModule implements OnModuleInit {
     connect(url, {useNewUrlParser: true, useUnifiedTopology: true, useCreateIndex: true, useFindAndModify: false});
 
     const onEvent = (_, eventMessage) => {
-      const event = plainToClass(sensorEventType.getType(eventMessage.eventType), eventMessage.data);
-      this.sensorProcessor.process(event);
+      if (eventMessage.metadata && eventMessage.metadata.originSync) {
```

Comment:
Ik kan niet zien waar `originSync` ontstaat. Ben je die vergeten toe te voegen in dit PR?

Reply:
Ah..